### ISSUE TITLE
[BUG] PMM | kraken loop on minimum order

### ISSUE THREAD

**Issue by PtrckM** [CoinAlpha/hummingbot#1668]
**Describe the bug**
- unable to pass minimum order using kraken connector

**Steps To Reproduce**
1. create pmm strategy 
2. choose kraken connector
3. choose token pair
4. enter spreads
5. enter order amount (min order) on loop

**Screenshots**
![image](https://user-images.githubusercontent.com/54516858/80307292-b0712400-87fa-11ea-8591-133d76c2489b.png)

**WORKAROUND**
- remove kraken api and keys before creating a pmm strategy conf
```
-rw-rw-r--  1 patrick patrick  441 Apr 26 12:07 encrypted_kraken_api_key.json
-rw-rw-r--  1 patrick patrick  505 Apr 26 12:07 encrypted_kraken_secret_key.json
```
- or copy other pmm strategy conf then edit then use import on bot

![image](https://user-images.githubusercontent.com/54516858/80307385-4efd8500-87fb-11ea-93da-f5f8be4bf88b.png)
![image](https://user-images.githubusercontent.com/54516858/80307387-5329a280-87fb-11ea-87de-33155a3a50fe.png)
![image](https://user-images.githubusercontent.com/54516858/80307398-718f9e00-87fb-11ea-8cad-0d7fb2e61c8a.png)

**Release version**
0.26.0 / Source Build, Docker Build and Binary Installer

**Comment by dennisocana** [comment]
Posting traceback from logs also to be used for referencing future related errors.

```
2020-04-29 07:31:37,136 - 17484 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: Unable to convert 'USDC' to 'XETH'. Aborting.
Traceback (most recent call last):
  File "hummingbot\core\utils\async_utils.py", line 9, in safe_wrapper
  File "hummingbot\client\command\create_command.py", line 64, in prompt_for_configuration
  File "hummingbot\client\command\create_command.py", line 129, in asset_ratio_maintenance_prompt
  File "hummingbot\user\user_balances.py", line 137, in base_amount_ratio
  File "hummingbot\core\utils\exchange_rate_conversion.py", line 179, in convert_token_value_decimal
  File "hummingbot\core\utils\exchange_rate_conversion.py", line 210, in convert_token_value
ValueError: Unable to convert 'USDC' to 'XETH'. Aborting.
```

**Comment by PtrckM** [comment]
Confirmed fix on dev-0.27.0

![image](https://user-images.githubusercontent.com/54516858/81272770-846a6400-9080-11ea-9a1b-5d7909a28cca.png)
![image](https://user-images.githubusercontent.com/54516858/81272834-9d731500-9080-11ea-91e4-23a1bd39b014.png)
